content/pt-br/containerization.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Contêinerização
+status: Completed
+category: tecnologia
+---
+
+## O que é
+A conteinerização é o processo de agrupar uma aplicação e suas dependências em uma [imagem de contêiner](/container-image/). O processo de criação do contêiner requer adesão ao padrão [Open Container Initiative](https://opencontainers.org) (OCI). Desde que a saída seja uma imagem de contêiner que atenda a esse padrão, qual a ferramenta de conteinerização usada não importa.
+
+## Problema relacionado
+Antes que o uso de contêineres se tornasse mais comum , as organizações usavam máquinas virtuais (VMs) para orquestrar várias aplicações em uma única [máquina baremetal](/bare_metal_machine/). As VMs são significativamente maiores que os contêineres e exigem um *hypervisor* para serem executados. Devido ao armazenamento, backup e transferência desses *templates* de VM maiores, a criação dos *templates* de VM também é lenta. Além disso, as VMs podem sofrer inconsistências nas configurações, o que viola o princípio da [imutabilidade](/immutable_infrastructure/).
```

Comment:
```suggestion
Antes que o uso de contêineres se tornasse mais comum , as organizações usavam máquinas virtuais (VMs) para orquestrar várias aplicações em uma única [máquina bare metal](/bare_metal_machine/). As VMs são significativamente maiores que os contêineres e exigem um *hypervisor* para serem executados. Devido ao armazenamento, backup e transferência desses *templates* de VM maiores, a criação dos *templates* de VM também é lenta. Além disso, as VMs podem sofrer inconsistências nas configurações, o que viola o princípio da [imutabilidade](/immutable_infrastructure/).
```

When searching I've seen baremetal machine being translated to `máquina bare metal` in most of the places - maybe we could go with this translation?